.changeset/great-lions-buy.md

```diff
@@ -0,0 +1,9 @@
+---
+'@xstate/test': major
+---
+
+@author: @mattpocock @davidkpiano
+
+Substantially simplified how paths and plans work in `TestModel`. Changed `getShortestPlans` and `getSimplePlans` to `getShortestPaths` and `getSimplePaths`. These functions now return an array of paths, instead of an array of plans which contain paths.
+
+Also added `getPaths`, which defaults to `getShortestPaths`.
```

Comment:
This sounds a little bit like if `getPaths` is redundant. Maybe worth mentioning here that it allows you to customize the `pathGenerator`? I would probably even remove `getShortestPaths` from being its default there - it's nice to surface what `pathGenerator` is actually used and if somebody wants to use shortest paths then they should just use `getShortestPaths`

Reply:
I disagree, I often struggle to find use cases for getSimplePaths and I think most users will just use getPaths without needing to worry about path generation. I like having the alias.

Reply:
Moving this to Discord, it's got wider implications than this PR